source/transactions/tests/auto-start.yml

```diff
@@ -273,70 +273,6 @@ tests:
         data:
           - _id: 2
 
-  - description: commit empty transaction
-
-    sessionOptions:
-      session0:
-        autoStartTransaction: true
-
-    operations:
-      # Commit before any operations.
-      - name: commitTransaction
-        arguments:
-          session: session0
-      - name: insertOne
-        arguments:
-          document:
-            _id: 1
-          session: session0
-        result:
-          insertedId: 1
-      - name: commitTransaction
-        arguments:
-          session: session0
-      # Commit again with no active transaction.
-      - name: commitTransaction
-        arguments:
-          session: session0
-        result:
-          errorContains: no transaction started
```

Comment:
I agree this test is broken. According to the spec, the last commitTransaction should succeed without running a commitTransaction command. Can you just remove the error result here?

Reply:
SGTM